### PR TITLE
Fundamental Concepts - CorDapps

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/platform/corda/5.0/key-concepts/fundamentals/CorDapps/_index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+Execution and enforcement of these rules are determined by the underlying model chosen for the system of record. Public blockchains and cryptocurrency networks use different validation and consensus mechanisms than the Corda UTXO ledger, yet the underlying principle of allowing untrusting entities to propose updates and have them accepted into the global record as long as they conform to the rules and are agreed to the parties is true across all decentralized systems.
+
+### Affirmative Consent
+
+In parallel to the concept of validity, Corda also addresses the concept of affirmation. That is, the ability for an entity to signify, irrefutably to all, that it consents to a proposal in some capacity (for example, as a benefactor or observer). 
+
+This is acheived through a cryptographic signature on a hash of the proposal. A set of signatures by all involved parties is sufficient to indicate that all agree that the update should take place. In conjunction with the execution of the validity rules, Corda can ensure the global state is only changed in ways that are valid and trustable by all.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0/key-concepts/fundamentals/CorDapps/_index.md:143]:
```suggestion
This is achieved through a cryptographic signature on a hash of the proposal. A set of signatures by all involved parties is sufficient to indicate that all agree that the update should take place. In conjunction with the execution of the validity rules, Corda can ensure the global state is only changed in ways that are valid and trustable by all.
```

**Review by AnnaK-R3 — APPROVED** [review]

File: content/en/platform/corda/5.0/key-concepts/fundamentals/CorDapps/_index.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+	 figcaption="Centralized Versus Decentralized Agreement"
+>}}
+
+In a centralized system, how can an entity trust the global state when it has no direct say in all of the mutations applied?
+
+{{< 
+  figure
+	 src="trust.png"
+   width="50%"
+	 figcaption="Issue of Trust in Centralized Agreements"
```
Comment (yarivus) [inline content/en/platform/corda/5.0/key-concepts/fundamentals/CorDapps/_index.md:117]: should be `Decentralized`

File: content/en/platform/corda/5.0/key-concepts/fundamentals/CorDapps/_index.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+Smart contracts can have different levels of granularity (network-wide or individual data representations) but the same general principles hold true that a proposal is only valid when the smart contract says it is. Rules in Corda can be arbitrary; however, there is a strong requirement that validity controls are deterministic as once evaluated as valid, checks on a change in the future must always return that it is valid. This means smart contracts can only reason about data present within the system of record: anything outside that generally leads to non-deterministic results as their presence cannot be guaranteed. 
+
+{{< 
+  figure
+	 src="proposal-rules.png"
+   width="50%"
+	 figcaption="Validity of Proposals"
+>}}
+
+It is the responsibility of every party to a proposal to check that it is valid. This is a trustless environment where it is possible for a counterparty to submit a proposal that is not valid by the rules. Accepting an invalid update could have a number of consequences, including financial and reputational.
```
Comment (yarivus) [inline content/en/platform/corda/5.0/key-concepts/fundamentals/CorDapps/_index.md:135]: should be `negative consequences`

File: content/en/platform/corda/5.0/key-concepts/fundamentals/CorDapps/_index.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+Execution and enforcement of these rules are determined by the underlying model chosen for the system of record. Public blockchains and cryptocurrency networks use different validation and consensus mechanisms than the Corda UTXO ledger, yet the underlying principle of allowing untrusting entities to propose updates and have them accepted into the global record as long as they conform to the rules and are agreed to the parties is true across all decentralized systems.
```
Comment (yarivus) [inline content/en/platform/corda/5.0/key-concepts/fundamentals/CorDapps/_index.md:137]: should be `agreed by the parties`

**Review by yarivus — COMMENTED** [review]
@nadinequinr3 left some comments

**Pull request merged**